### DISCUSSION TITLE
Does unocss support the differentiation of nested groups in `group-{modifier}`?

### DISCUSSION THREAD

**Discussion by hakadao** [unocss/unocss#2348] [Q&A] [answered]
In tailwindcss, this feature refers to a specific parent group when using nested groups.
Does unocss support this feature? Or is there an alternative way to do it?
https://tailwindcss.com/docs/hover-focus-and-other-states#differentiating-nested-groups

**Comment by chu121su12** [comment, marked as answer]
The `<pseudo>/<label>` was added as a _"beta"_ not long ago.

https://github.com/unocss/unocss/blob/5d472fac9be486958fbf5812dbedef6ffe84fe5e/test/assets/preset-mini-targets.ts#L1035-L1039

**Reply by hakadao** [reply]
Thanks.  I hope to see this feature implemented in the official version.😀
